src/components/TransactionHistory/TransactionTypeIcon.tsx

```diff
@@ -13,7 +13,7 @@ const IconWrapper: React.FC<CenterProps> = props => (
   <Center borderRadius='full' boxSize='100%' {...props} />
 )
 
-export const TransactionTypeIcon = ({ type, status }: { type: string; status: TxStatus }) => {
+export const TransactionTypeIcon = ({ type, status }: { type?: string; status?: TxStatus }) => {
```

Comment:
The `switch` statement below checks with a final default case mean these can both be optional. Indeed, at most one of `type` and `status` is ever used.